server/api_archive.go

```diff
@@ -0,0 +1,126 @@
+package server
+
+// This is a low-level archiver which zips and sends GCA server persistent file
+// data using a POST command. This is intended to be used by an external service
+// to provide archival snapshots of raw data in the system. Only public data is
+// provided, with private keys zeroed before sending.
```

Comment:
Should the private keys be included at all?

Reply:
I wanted to keep the format of this file the same, as it already contains an extra 32 zero bytes. I don't have a really good reason for this except "change as little as possible".